docs/contributors/code/react-native/osx-setup-guide.md

```diff
@@ -71,28 +40,46 @@ npm run distclean
 npm ci
 ```
 
-## Unit Tests
+## iOS
 
-Unit tests should work at this point.
+### CocoaPods
 
-```sh
-npm run native test
+[CocoaPods](https://guides.cocoapods.org/using/getting-started.html) is needed to fetch React and third-party dependencies:
+
+```
+sudo gem install cocoapods
 ```
 
-## iOS
+Note, Mac M1 is not directly compatible with Cocoapods. If you encounter issues, try running these commands to install the ffi package, which will enable pods to be installed with the proper architecture:
 
-The easiest way to figure out what needs to be installed is using the
-[react native doctor](https://reactnative.dev/blog/2019/11/18/react-native-doctor). From your checkout, or
-relative to `/packages/react-native-editor folder`, run:
+```
+sudo arch -x86_64 gem install ffi
```

Comment:
Similar to my [comment above](https://github.com/WordPress/gutenberg/pull/40964/files#r872416814), my impression is that one doesn't need to manually install or set the architecture for the `ffi` gem when using `rbenv` and Ruby `v2.7.4` (not `v3.x.x`).

Reply:
@hypest, ah, I was hoping that [the comment here](https://github.com/WordPress/gutenberg/pull/40964#discussion_r873833823) addressed that. The comment about using `ffi` is now only under the `System Ruby` section, for people who may be using the default version of Ruby (for extra context, this was taken straight from [the React Native docs](https://reactnative.dev/docs/environment-setup)). I've now added an explicit mention that it may not need to be manually installed if using a Ruby manager like `rbenv` in https://github.com/WordPress/gutenberg/pull/40964/commits/a2dd84e9b3faec47d6bf9b4dfc093128a9911894 now. How does that seem?

Reply:
Ah, gotcha. Thanks for the additional commit to make it more explicit. I think it's good to go!